index.bs

```diff
@@ -3556,17 +3552,139 @@ A [=remote end=] has a <dfn>before request sent map</dfn> which is initially an
 empty map.  It's used to track the network events for which a
 <code>network.beforeRequestSent</code> event has already been sent.
 
+### Network Intercepts ### {#network-intercepts}
+
+A <dfn>network intercept</dfn> is a mechanism to allow remote ends to intercept
+and modify network requests and responses.
+
+A [=BiDi session=] has an <dfn>intercept map</dfn> which is a [=/map=] between
+intercept id and a [=tuple=] with fields <code>url patterns</code> and
```

Comment:
```suggestion
intercept id and a [=struct=] with fields <code>url patterns</code> and
```

Why struct instead of tuple? Because `addIntercept `step #6 is referring to this as "struct".

(The other way around also works, but in this case the order seems to not matter, so `struct` seems a better fit.)